.github/workflows/test.yml

```
@@ -461,6 +461,18 @@ jobs:
         env:
           ZEBRA_CONTAINER: ${{ steps.get-container-name.outputs.zebra_container }}
 
+      - name: Check exit code
+        id: check-exit-code
+        run: |
+          gcloud compute ssh \
+          sync-checkpoint-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }} \
+          --zone ${{ env.ZONE }} \
+          --quiet \
+          --ssh-flag="-o ServerAliveInterval=5" \
+          --command="exit_code=$(docker wait ${{ env.ZEBRA_CONTAINER }}) && if [ $exit_code -eq "0" ]; then exit 0; else exit 1; fi"
```

Question: is there any reason to turn all non-zero edit codes into 1?

```suggestion
          --command="docker wait ${{ env.ZEBRA_CONTAINER }}"
```

---

This was my first try, but I stumbled with this issue https://github.com/docker/cli/pull/2244#issue-546334620

---

Wow, that's a terrible interface, sorry you have to deal with that!

---

Maybe we could put a comment in the file, so we don't accidentally simplify the code and break it?